develop/learning-paths/migration/articles/xx-theme/04-updating-the-resources-importer.markdown

```diff
@@ -0,0 +1,365 @@
+                    },
+                    "tip": {
+                        "en_US": ""
+                    },
+                    "dataType": "html",
+                    "fieldNamespace": "ddm",
+                    "indexType": "keyword",
+                    "localizable": true,
+                    "name": "content",
+                    "readOnly": false,
+                    "repeatable": false,
+                    "required": false,
+                    "showLabel": true,
+                    "type": "ddm-text-html"
+                }
+            ]
+        }     
+
+    This structure simply identifies the language and field settings for the 
+    articles, and specifies a `name` to identify the content. In previous 
+    versions of Liferay, article structures were XML, now they must be JSON. Now 
+    that you have the structure created, you can create the matching template 
+    next.
+
+5.  Create a `Basic Web Content.ftl` template file in the 
+    `resources-importer/journal/templates/Basic Web Content/`
+    folder you just created and add the following code to it:
+
+        ${content.getData()}
+
+This accesses the article's content by the `name` defined in the structure, 
+`content` in this case, and renders the HTML data. Now that you have the basic
+web content structure and template created, all that's left to update is the
+article type.
+
+### Updating the Web Content Articles to XML
+
+In 6.2, HTML articles were an acceptable type for the resources importer. In
+Liferay 7, all articles must be of type XML. This migration is easier then it
+may sound for basic web content. You just need to wrap the HTML in 
+`<![CDATA[...]]>` tags.
+
+For example, to update the `2 column description.html` article to XML, you would
+write the following in a `2 column description.xml` file:
+
+    <?xml version="1.0"?>
+
+    <root available-locales="en_US" default-locale="en_US">
+	    <dynamic-element name="content" type="text_area" 
+	    index-type="keyword" index="0">
+		    <dynamic-content language-id="en_US">
+			    <![CDATA[
+				    <div class="container-fluid">
+	    <div class="col-md-4" id="columnLeft">Out of This World</div>
+	    <div class="col-md-8" id="columnRight">Come to the Lunar Resort and 
+	    live out your childhood dream of being an astronaut on the Moon. If 
+	    that's not enough incentive, you'll enjoy a luxurious 3 day 2 night 
+	    stay in our fabulous Lunar accomodations. Enjoy a round of Lunar 
+	    Golf on our one of a kind course. Have a blast on our Rover Racing 
+	    track. Make your reservation now. The rest of your life starts 
+	    today!</div>
+    </div>
+    ]]>
+		    </dynamic-content>
+	    </dynamic-element>
+    </root>
+
+You may have also noticed that the `<div>` classes have been changed. The 
+`2 column description.xml` article shown above illustrates the updated Bootstrap 
+grid system for  Bootstrap 3. Instead of using the `span-[number]` class to 
```

Comment:
The bootstrap 2.x classes didn't have the dash. For example, `span3`. Obviously not a huge deal since they will be migrating away from these classes, but in case seeing the dash throws them off.